#! /bin/sh /usr/share/dpatch/dpatch-run
## man-refs.dpatch by Matej Vela <vela@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Ensure that all man pages have a consistent set of references
## DP: (#294805, #309663).

@DPATCH@
diff -urNad wv-1.2.1~/help/man/wvAbw.1 wv-1.2.1/help/man/wvAbw.1
--- wv-1.2.1~/help/man/wvAbw.1	2006-03-18 17:32:57.000000000 +0100
+++ wv-1.2.1/help/man/wvAbw.1	2006-03-18 17:39:07.000000000 +0100
@@ -22,6 +22,7 @@
 .BR wvPS(1),
 .BR wvDVI(1),
 .BR wvPDF(1),
+.BR wvRTF(1),
 .BR wvText(1),
 .BR wvWml(1),
 .BR wvMime(1),
diff -urNad wv-1.2.1~/help/man/wvCleanLatex.1 wv-1.2.1/help/man/wvCleanLatex.1
--- wv-1.2.1~/help/man/wvCleanLatex.1	2006-03-18 17:32:57.000000000 +0100
+++ wv-1.2.1/help/man/wvCleanLatex.1	2006-03-18 17:39:07.000000000 +0100
@@ -23,6 +23,7 @@
 .BR wvText(1),
 .BR wvWml(1),
 .BR wvMime(1),
+.BR wvRTF(1),
 .BR catdoc(1), 
 .BR word2x(1)
 .SH "AUTHOR"
diff -urNad wv-1.2.1~/help/man/wvDVI.1 wv-1.2.1/help/man/wvDVI.1
--- wv-1.2.1~/help/man/wvDVI.1	2006-03-18 17:32:57.000000000 +0100
+++ wv-1.2.1/help/man/wvDVI.1	2006-03-18 17:39:07.000000000 +0100
@@ -22,6 +22,7 @@
 .BR wvCleanLatex(1),
 .BR wvPDF(1),
 .BR wvPS(1),
+.BR wvRTF(1),
 .BR wvText(1),
 .BR wvWml(1),
 .BR wvMime(1),
diff -urNad wv-1.2.1~/help/man/wvHtml.1 wv-1.2.1/help/man/wvHtml.1
--- wv-1.2.1~/help/man/wvHtml.1	2006-03-18 17:32:57.000000000 +0100
+++ wv-1.2.1/help/man/wvHtml.1	2006-03-18 17:39:07.000000000 +0100
@@ -21,6 +21,7 @@
 .BR wvPS(1),
 .BR wvDVI(1),
 .BR wvPDF(1),
+.BR wvRTF(1),
 .BR wvText(1),
 .BR wvWml(1),
 .BR wvMime(1),
diff -urNad wv-1.2.1~/help/man/wvSummary.1 wv-1.2.1/help/man/wvSummary.1
--- wv-1.2.1~/help/man/wvSummary.1	2006-03-18 17:32:57.000000000 +0100
+++ wv-1.2.1/help/man/wvSummary.1	2006-03-18 17:39:24.000000000 +0100
@@ -19,6 +19,7 @@
 .BR wvHtml(1),
 .BR wvLatex(1),
 .BR wvCleanLatex(1),
+.BR wvPS(1),
 .BR wvPDF(1),
 .BR wvDVI(1),
 .BR wvMime(1),
diff -urNad wv-1.2.1~/help/man/wvText.1 wv-1.2.1/help/man/wvText.1
--- wv-1.2.1~/help/man/wvText.1	2006-03-18 17:32:57.000000000 +0100
+++ wv-1.2.1/help/man/wvText.1	2006-03-18 17:39:07.000000000 +0100
@@ -23,6 +23,7 @@
 .BR wvDVI(1),
 .BR wvPDF(1),
 .BR wvRTF(1),
+.BR wvWml(1),
 .BR wvMime(1),
 .BR catdoc(1), 
 .BR word2x(1)
